packages/messages-react/README.md

```diff
@@ -0,0 +1,86 @@
+# [`@fremtind/jkl-messages-react`](https://fremtind.github.io/jokul/komponenter/messages)
+
+## Beskrivelse
+
+Se portalen for [bruk og prinsipper](https://fremtind.github.io/jokul/komponenter/messages).
+
+## Kom i gang
+
+[Lær hvordan du kan ta i bruk Jøkul](https://fremtind.github.io/jokul/developer/getting-started/)
+
+## Bruk av React-pakken
+
+### Installasjon
+
+1. Installér pakken med `yarn add @fremtind/jkl-messages-react` eller `npm i @fremtind/jkl-messages-react`. Stil-pakken blir automatisk installert som en avhengighet.
+
+2. Importer _både_ React-pakken (velg komponenten(e) du trenger) og stil-pakken i prosjektet ditt:
+
+```js
+import { MessageBox, SystemMessage } from "@fremtind/jkl-messages-react";
+import "@fremtind/jkl-messags/message-box.min.css";
+import "@fremtind/jkl-messags/system-message.min.css";
+```
+
+### Bruk
+
+Pakken `jkl-messages-react` inneholder et sett med meldingskomponenter:
+
+-   `MessageBox`
+-   `SystemMessage`
+
+#### MessageBox
+
+Komponenten tar følgende props:
+
+-   `messageType`: **Påkrevd**. Meldingsboks type. `string`
```

Comment:
Kan vi være så snille å slutte å dokumentere props manuelt? It`s 2020 guys, tools can do this.

Reply:
Helt enig, kan du lage et issue og eventuelt foreslå noen verktøy om det finnes noe good shit?

Reply:
Hva med typescript? :p